code/modules/power/apc.dm

```diff
@@ -1244,24 +1244,28 @@
 			equipment = autoset(equipment, 0)
 			lighting = autoset(lighting, 0)
 			environ = autoset(environ, 0)
-			area.poweralert(0, src)
+			if(!integration_cog)
+				area.poweralert(0, src)
```

Comment:
Might be wise to wrap area.poweralert in some apc level method (so we only have to check the condition once!!!1!)

Reply:
Got it